src/client/opamClient.ml

```diff
@@ -1148,20 +1152,31 @@ let rec cygwin_menu ~bypass_checks header =
         end
     end
   in
-  Lazy.force header;
-  OpamConsole.msg
-    "\n\
-     opam and the OCaml ecosystem in general require various Unix tools in \
-     order to operate correctly. At present, this requires the installation \
-     of Cygwin to provide these tools.\n\n";
-  match OpamConsole.menu "How should opam obtain Unix tools?"
-          ~default ~no:default ~options with
+  let result =
+    if interactive then begin
+      Lazy.force header;
```

Comment:
Maybe keep the header outside the interactive ? Like that it give more information about the future notes.

Reply:
Yes - nice idea!